src/main/java/cards/CardChecker.java

```diff
@@ -32,6 +35,7 @@ public CardCombination getCombination(List<CardModel> cardModels) {
         Set<Map.Entry<CardModel, Long>> groupedCardModelEntries = groupedCardModel.entrySet();
 
         switch (groupedCardModelEntries.size()) {
+            // CR: Please move to constants that describe what each case represents
```

Comment:
// Answer: Its just a quantity of unique cards in set. There is no matter to create constants for this case

Reply:
In that case, THREE_UNIQUE_CARDS_IN_HAND could be a good name for the constant.
I understood your algorithm but sometimes when it becomes more complex and people add many things to it starts to get less and less readable.

Reply:
I get your point. Agree